react-bootstrap/react-bootstrap.d.ts

```diff
@@ -704,6 +704,88 @@ declare module "react-bootstrap" {
     }
     var FormControls: FormControlsClass;
 
+    // <Form />
+    // ----------------------------------------
+    interface FormProps extends React.HTMLAttributes {
+        bsClass?: string;
+        componentClass?: string;
```

Comment:
Yeah. I would like to use ReactType, but in another interfaces (like ButtonProps) there's a `componentClass` property with type `string`. Others with `any` type. I just follow the "pattern". 
There's several TODOs comments in the file. We need to fix this.